### DISCUSSION TITLE
Cannot understand the image uploaded or any attachments

### DISCUSSION THREAD

**Discussion by paulwongx** [open-webui/open-webui#1834] [General]
Maybe I'm misunderstanding this feature but when I upload an image and ask the AI to tell me what the text in the image says, it says the image is not detected. 

This is the response I get:
```
[img-0] is not an actual image, but rather a placeholder or a syntax error. 
It seems that you're trying to reference an image file, but it doesn't exist.
So, there is no text to display in this attachment because it's not a real image!
```

I guess under the hood the image isn't (can't?) be sent to Ollama to understand but if that's the case, what is the proper way to use the attachment feature?

If it should work, what am I doing wrong?

**Comment by silentoplayz** [comment]
https://docs.openwebui.com/tutorial/images should be able to help you out here. Getting a model to describe an image only works with multi-modal LLMs that support images as input.

**Reply by paulwongx** [reply]
Thanks for your response. This is related to image generation. My broader question is that any file I upload isn't recognized when using Open-webUI with Ollama. Not sure if I'm misunderstand the use case of the file upload, or if I'm doing something wrong, or
 maybe it's just me.

**Reply by silentoplayz** [reply]
As for your broader question about file uploads not being recognized when using Open WebUI with Ollama, it's possible that there are some configuration or setup issues at play. Without more information, it's hard to pinpoint the exact problem. However, I'd recommend checking the Open WebUI documentation and tutorial pages for guidance on using file uploads with your chosen LLM.

[Image Generation Mini Tutorial](https://docs.openwebui.com/assets/images/tutorial_image_generation-1034f021c1ab818750e4eb7a288c1823.png)

To guide you a little more towards being on the right track, here are quite a few models available for download on Ollama.com, that are each capable of accepting images as input and being able to describe such image(s) back to you:

https://ollama.com/library/llava
https://ollama.com/library/bakllava
https://ollama.com/library/llava-llama3
https://ollama.com/library/llava-phi3
https://ollama.com/library/moondream
